runtime/vm/jvminit.c

```diff
@@ -3955,6 +3955,31 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 	}
 
 	vm->checkpointState.lastRestoreTimeMillis = -1;
+
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+	{
+		IDATA xxCRaCCheckpointToIndex = FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, VMOPT_XXCRACCHECKPOINTTO, NULL);
+		if (xxCRaCCheckpointToIndex >= 0) {
+			PORT_ACCESS_FROM_JAVAVM(vm);
+			if (J9_ARE_ANY_BITS_SET(vm->checkpointState.flags, J9VM_CRIU_IS_CHECKPOINT_ENABLED)) {
+				/* -XX:+EnableCRIUSupport was specified which is incompatible with CRaC */
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_VM_CRIU_CRAC_INCOMPATIBLE_SETTING, NULL);
+				return JNI_ERR;
+			} else {
+				char *cracCheckpointToValue = NULL;
+
+				optionValueOperations(PORTLIB, vm->vmArgsArray, xxCRaCCheckpointToIndex, GET_OPTION, &cracCheckpointToValue, 0, '=', 0, NULL);
+				Trc_VM_crac_checkpointTo(cracCheckpointToValue);
+				vm->checkpointState.cracCheckpointToDir = cracCheckpointToValue;
+
+				/* just like -XX:+EnableCRIUSupport was specified */
+				vm->checkpointState.flags |= J9VM_CRIU_IS_CHECKPOINT_ENABLED | J9VM_CRIU_IS_CHECKPOINT_ALLOWED;
```

Comment:
Actually, I think we need a new flag. `J9VM_CRAC_IS_CHECKPOINT_ENABLED`

Reply:
`InternalCRIUSupport::isCRIUSupportEnabledImpl` should return true if either is set.

Reply:
CRIUSupport::isCRIUSupportEnabled should only return true if `J9VM_CRIU_IS_CHECKPOINT_ENABLED` is set.

Reply:
Added a new flag `J9VM_CRAC_IS_CHECKPOINT_ENABLED`, and updated methods related to `J9VM_CRIU_IS_CHECKPOINT_ENABLED`.